providers/AppServiceProvider.js

```diff
@@ -23,6 +23,12 @@ class AppServiceProvider extends ServiceProvider {
 
       return new LotJobService(JobService);
     });
+
+    ioc.bind('BidWinnerService', (app) => {
```

Comment:
this.app.bind('BidWinnerService', (app) => {

Reply:
+